documentation/docs/examples/point-to-point-measurement.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: point-to-point-measurement
+title: Point to Point Measurement
+description: Demo shows how to measure point to point distance in Reveal.
+---
+
+import { DemoWrapper } from '@site/docs/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The `MeasurementTool` is a measuring tool for point to point distance with the 3D objects.
+This can be achieved by selecting a point on the 3D Object and drag the pointer to the destined point to get measurement of the distance.
+The tools default measurement is in meters, but the tool provides user to customise the measuring units.
+
+The tool provides ability to change measuring line width & color.
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+## Point to point distance measurment
+To enter into point to point measurement mode, use `MeasurementTool.enterMeasurementMode()` which will have default line width & color.
```

Comment:
```suggestion
To enter point-to-point measurement mode, use `MeasurementTool.enterMeasurementMode()`. This will register events for handling mouse clicks and touch events to allow placing the measurement end-points.
```